src/odemis/gui/comp/viewport.py

```diff
@@ -42,9 +43,11 @@
 from odemis.gui.util import call_in_wx_main
 from odemis.gui.util.raster import rasterize_line
 from odemis.model import NotApplicableError
-from odemis.util import units, spectrum, peak
+from odemis.util import units, spectrum, peak, img
 import wx
 from odemis.util import no_conflict
+from odemis.acq.stream._projection import TemporalSpectrumProjection, \
```

Comment:
Import directly from `odemis.acq.stream` (not the hidden `._projection`).
Probably, only `DataProjection` is needed anyway.